client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -158,6 +161,9 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
         fetchAllInvestigationStatuses();
         fetchAllDesksByCountyId();
         startWaiting();
+        WindowTabsBroadcatChannel.current.onmessage = function (ev) { 
```

Comment:
ev stands for event?

Reply:
yes

Reply:
so maybe change the name to event

Reply:
I think it's clear enough, but changed it anyway